jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/ui/utils/ViewExtensions.kt

```diff
@@ -0,0 +1,5 @@
+import android.content.Context
+import android.view.View
+
+val View.ctx: Context
```

Comment:
You are right, but I think the point was to introduce some simple extension to show how it works :) I will just remove it again!